lint-staged.config.js

```diff
@@ -1,4 +1,11 @@
 module.exports = {
-  "*.{js,ts}": 'eslint --ignore-pattern "**/build-scripts/**/*.js" --fix',
-  "!(/translations)*.{js,ts,json,css,md,html}": "prettier --write",
+  "*.{js,ts}": [
+    "prettier --write",
+    'eslint --ignore-pattern "**/build-scripts/**/*.js" --fix',
+  ],
+  "!(/translations)*.{json,css,md,html}": "prettier --write",
+  "translations/*/*.json": (files) =>
```

Comment:
I think that most people that work on translations probably edit via the GitHub UI. It might be better to have a GitHub action that leaves a comment on PRs that do.

Reply:
Yup, that's next